src/settings.cpp

```diff
@@ -225,6 +225,16 @@ void ShotcutSettings::setViewMode(const QString& viewMode)
     emit viewModeChanged();
 }
 
+QString ShotcutSettings::snapSuffix() const
+{
+    return settings.value("snapSuffix", ".png").toString();
```

Comment:
I do not like the key name "snap" I had to stop and think for many seconds about what this is until I realize "snapshot." "Snapshot" is kinda OK except we do not have anything called snapshot. `exportFrameSuffix`

Reply:
Done